lib/rubocop/cop/layout/space_inside_parens.rb

```diff
@@ -86,6 +119,14 @@ def each_extraneous_space(tokens)
           end
         end
 
+        def each_extaneus_space_between_right_parents(token1, token2)
```

Comment:
```suggestion
        def each_extraneous_space_between_right_parents(token1, token2)
```

Or just use `extra`?